src/render/texture.js

```diff
@@ -40,12 +40,14 @@ class Texture {
     format: TextureFormat;
     filter: ?TextureFilter;
     wrap: ?TextureWrap;
+    powerOfTwoSize: boolean;
 
     constructor(context: Context, image: TextureImage, format: TextureFormat, premultiply: ?boolean) {
         this.context = context;
 
         const {width, height} = image;
         this.size = [width, height];
+        this.powerOfTwoSize = width === height && (Math.log(width) / Math.LN2) % 1 === 0;
```

Comment:
Is there a reason to use state rather than compute this as needed? As state it could be wrong if the size is changed via `update`.

Reply:
BTW a common neat and fast way to check if a number is a power of two is `x & (x - 1) === 0`